example/cumulus-tf/main.tf

```diff
@@ -66,6 +66,8 @@ data "aws_ecr_repository" "async_operation" {
 
 module "cumulus" {
   source = "../../tf-modules/cumulus"
+
+  lambda_timeouts = var.lambda_timeouts
```

Comment:
Where did this come from? Might be right but I'm surprised to see something other than version bumps and docs.

Reply:
Nevermind, looks like you removed it.

Reply:
Right you are, I pushed up a change that removed it, sorry about that.